### DISCUSSION TITLE
夜爬泰山 | /posts/taishan/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#582] [Announcements]
## 夜爬泰山

https://yuanfan.rbind.io/posts/taishan/

**Comment by CyrusYip** [comment]
> 我想不到怎么把那种被石头“居屁股”“居背”的那种“居法”用普通话或者更书面的语言表达，也不是扎，也不是刺，因为受力点并不是只有一个点

你说的是「硌」？我查了「居」和「居背」，没查到啥意思，这是方言吗？

>  硌
>
> 碰到堅硬不平的東西而引起損傷或痛苦。清．蒲松齡《富貴神仙》第三回：「自己鋪下草，找了一塊半頭磚，嫌硌頭，又使衣服墊。」
>
> https://dict.revised.moe.edu.tw/dictView.jsp?ID=3828

**Reply by earfanfan** [reply]
啊，是的，可以说“硌”，我们老家方言里面既有“硌人”也有“居人”，我原以为“硌”和“居”都是方言呢，原来“硌”不是。
